omics_orientation/omics_orientation.md

```diff
@@ -0,0 +1,195 @@
+<!--
+
+author:   Meredith Lee
+email:    leemc@chop.edu
+version:  1.0.0
+module_template_version: 2.0.0
+language: en
+narrator: UK English Female
+title: Omics Orientation
+comment: This module provides a brief introduction to omics and its associated fields.
+long_description: Omics is a wide-reaching field, with many different subfields. This module aims to disambiguate several omics-related terms and topics, discuss some of the most popular omics research fields, and examine the challenges of and caveats for omics research.
+estimated_time: 15m
+
+@learning_objectives
+
+After completion of this module, learners will be able to:
+
+- Define what omics is and explain why a researcher might choose an omics approach
+- Identify several popular omics domains
+- Describe some challenges and caveats of omics research
+
+
+@end
+
+link:  https://chop-dbhi-arcus-education-website-assets.s3.amazonaws.com/css/styles.css
+
+script: https://kit.fontawesome.com/83b2343bd4.js
+
+-->
+
+# Omics Orientation
+
+
+<div class = "overview">
+
+## Overview
+@comment
+
+**Is this module right for me?** @long_description
+
+**Estimated time to completion:** @estimated_time
+
+**Pre-requisites:** None
+
+**Learning Objectives**
+
+@learning_objectives
+
+
+</div>
+
+
+## What is "Omics"?
```

Comment:
General question: should omics be in quotes? I think maybe just when defining it and giving a pronunciation, but if it does need to be in quotes, then it should have it in quotes everywhere.

Reply:
I actually think it should be here. I think it should really only be in quotes when I'm talking about it as a suffix. Does that seem reasonable?

Reply:
Very reasonable!